src/tictactoe_clojure/io.clj

```diff
@@ -0,0 +1,9 @@
+(ns tictactoe-clojure.io)
```

Comment:
The name `io` is a bit too general for this namespace. It is a type of IO, but it's console specific. It might be a good idea to rename this.